### PR TITLE
Improve Project

### PR DESCRIPTION
**Pull request by mooziii**
Switched from Maven to Gradle
Renamed Main class
Delete empty and unused lines

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+plugins {
+    java
+    kotlin("jvm") version "1.4.21"
```
Comment (NyCodeGHG) [inline build.gradle.kts]: Why are here kotlin plugins?

Comment (DRSchlaubi) [inline build.gradle.kts]: Why Kotlin the lang is amazing yes but it's unused here LOL
Also Kotlin 1.4.21?? we're at 1.4.31 now

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-bin.zip
```
Comment (DRSchlaubi) [inline gradle/wrapper/gradle-wrapper.properties]: 6.8.3 :D

Comment (NyCodeGHG) [inline gradle/wrapper/gradle-wrapper.properties]: I think gradle-6.8.3-all.zip would be better

File: src/main/java/tv/banko/tutorialreloaded/utils/Config.java [hunk 3]
```diff
@@ -12,21 +12,17 @@
```
Comment (DRSchlaubi) [inline src/main/java/tv/banko/tutorialreloaded/utils/Config.java:12]: Why not replace the whole class with `Plugin#getConfig()` wouldn't make a difference

**Review by DRSchlaubi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**